test/general.test.js

```diff
@@ -167,15 +167,21 @@ describe('Read theme', function () {
         readTheme(themePath('partials')).then(function (theme) {
```

Comment:
This test read's a theme which is called `partials`, which contains a folder `partials`. Somehow confusing.
I know this is not part of your task here, but could you please rename this folder to e.g. `theme-with-partials`?